lib/iris/fileformats/grib/__init__.py

```diff
@@ -900,32 +893,26 @@ def load_cubes(filenames, callback=None, regularise=True, auto_regularise=None):
         information. The default behaviour is to interpolate cubes on a
         reduced grid to an equivalent regular grid.
 
-        .. deprecated:: 1.8. Please use the `regularise` kwarg instead.
-
-    .. note::
-
-       To make use of the *regularise* keyword the normal Iris loading
-       pipeline cannot be used, the loading must be performed manually::
+        .. deprecated:: 1.8. Please use strict_grib_load and regrid instead.
+        
```

Comment:
This blank line is redundant (and has trailing whitespace) - please could you chop it out.